config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml

```diff
@@ -7,7 +7,7 @@ presets:
     # remove this after https://github.com/kubernetes/kubernetes/pull/69051
   - name: KUBE_CUSTOM_NETD_YAML
     # we want to keep extra spaces for the yaml file.
-    value: |2+
+    value: |2
```

Comment:
What’s going on here?

Reply:
I think that's a [block-chomping indicator ](https://yaml-multiline.info/) that's being automatically removed, since it's chomping no lines